frontend/src/styles/color-variables.sass

```diff
@@ -8,3 +8,11 @@ $grey: #777777
 $light-copper: #F5A75F
 $dark-grey: #121415
 $light-grey: #1D1E21
+
+$main-background: #121415
+$element-background: #222327
+$main-font-color: #FFFFFF
+$link-color: #705FF5
+$border-color: #FFFFFF
+$button-enabled: #61AD00
+$button-disabled: #325602
```

Comment:
please avoid color duplications
most of these colors already exist, so reuse them

and it would be better to name colors as their color name. not by element where we apply them

Reply:
Done as you suggested